packages/frontend/components/Borrow/LTVProgressBar.tsx

```diff
@@ -32,21 +33,64 @@ export default function LTVProgressBar(props: LTVProgressBarProps) {
               sx={{
                 marginRight: "0.313rem",
                 fontSize: "0.875rem",
+                display: { xs: "none", sm: "inline" },
               }}
             />
           </Tooltip>
-          <Typography variant="xsmall">45% LTV (Recommended)</Typography>
+          <ClickableTooltip title="???" placement="top">
+            <InfoOutlinedIcon
+              sx={{
+                marginRight: "0.313rem",
+                fontSize: "0.875rem",
+              }}
+            />
+          </ClickableTooltip>
+          <Typography
+            variant="xsmall"
+            sx={{ display: { xs: "none", sm: "inline" } }}
```

Comment:
Parent component already have `display: { xs: "none" }`, are you sure it's needed here ?